packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -3,6 +3,8 @@ const _ = require(`lodash`)
 const urlJoin = require(`url-join`)
 import HttpAgent from "agentkeepalive"
 // const http2wrapper = require(`http2-wrapper`)
+const url = require(`url`)
+const querystring = require(`querystring`)
```

Comment:
Can you use [searchParams](https://nodejs.org/api/url.html#url_class_urlsearchparams) please.

Reply:
what's the difference?

Reply:
ah <img width="441" alt="Screen Shot 2021-08-23 at 12 31 21 PM" src="https://user-images.githubusercontent.com/71047/130508041-1c543322-7ce6-44b1-9662-5405ec3558f8.png">